xarray/tests/test_variable.py

```diff
@@ -2942,8 +2945,8 @@ def test_from_pint_wrapping_dask(self, Var):
         (np.array([np.datetime64("2000-01-01", "ns")]), False),
         (np.array([np.datetime64("2000-01-01", "s")]), True),
         (pd.date_range("2000", periods=1), False),
-        (datetime(2000, 1, 1), False),
-        (np.array([datetime(2000, 1, 1)]), False),
+        pytest.param(datetime(2000, 1, 1), True, marks=requires_pandas_3),
+        pytest.param(np.array([datetime(2000, 1, 1)]), True, marks=requires_pandas_3),
```

Comment:
With pandas 3, `pd.Series(datetime.datetime(...))` will produce a `Series` with `np.datetime64[us]` values instead of `np.datetime64[ns]`  values, so this conversion now warns.